app/views/admin/document_collection_groups/reorder.html.erb

```diff
@@ -0,0 +1,29 @@
+<% content_for :page_title, "Reorder groups for #{@collection.title}" %>
+<% content_for :title, "Reorder groups" %>
+<% content_for :context, @collection.title %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= form_with url: order_admin_document_collection_groups_path(@collection, @group), method: :put do |form| %>
+      <%= render "govuk_publishing_components/components/hint", {
+        text: "Use the up and down buttons to reorder pages, or select and hold on a page to reorder using drag and drop.",
+        margin_bottom: 4,
+      } %>
+      <%= render "govuk_publishing_components/components/reorderable_list", {
+        items: @collection.groups.map do |group|
+          {
+            id: group.id,
+            title: group.heading
+          }
+        end
+      } %>
+
+      <div class="govuk-button-group govuk-!-margin-bottom-6">
+        <%= render "govuk_publishing_components/components/button", {
+          text: "Save",
```

Comment:
Do we need to add Google Analytics to these buttons?

Reply:
Yes, good point. I've added that, I think the values are what would be expected.